api/src/main/java/com/flipkart/poseidon/internal/ParamValidationFilter.java

```diff
@@ -284,4 +285,24 @@ private List getValues(String[] values, ParamPOJO.DataType datatype) {
     private List<String> getStringValues(String[] values) {
         return Arrays.asList(values);
     }
+
+    private List getEnumValues(String[] values, String javaType) throws BadRequestException {
```

Comment:
Lets have a validator on javaType to be mandatory in case type is ENUM. (As discussed, in next PR)